capstone/src/main/resources/static/event.html

```diff
@@ -40,17 +37,17 @@ <h1>Create an Event </h1>
         <br><br>
 
         <label for="eventLocation"> Event Location: </label> <br>
-        <input type="number" id="eventLat" name="eventLat" placeholder="Event Latitude"> <br>
-        <input type="number" id="eventLng" name="eventLng" placeholder="Event Longitude">
+        <input type="number" step="0.000001" id="eventLatitude" name="eventLatitude" placeholder="Event Latitude"> <br>
+        <input type="number" step="0.000001" id="eventLongitude" name="eventLongitude" placeholder="Event Longitude">
```

Comment:
Works great! This is definitely out of the scope of this PR but we should consider adding like a mini map of some sort that takes a marker instead of having the user type in coordinates.